contracts/manifold/lazyclaim/ERC721LazyPayableClaim.sol

```diff
@@ -226,10 +258,19 @@ contract ERC721LazyPayableClaim is IERC165, IERC721LazyPayableClaim, ICreatorExt
         // Transfer funds
         _transferFunds(claim.erc20, claim.cost, claim.paymentReceiver, mintCount, claim.merkleRoot != "");
 
-        uint256[] memory newTokenIds = IERC721CreatorCore(creatorContractAddress).mintExtensionBatch(msg.sender, mintCount);
-        for (uint256 i; i < mintCount;) {
-            _tokenClaims[creatorContractAddress][newTokenIds[i]] = TokenClaim(uint224(claimIndex), uint32(newMintIndex+i));
-            unchecked { ++i; }
+        if (claim.contractVersion >= 3) {
+            uint80[] memory tokenData = new uint80[](mintCount);
```

Comment:
Can we rename this to `tokenDatas` just to maintain the plural naming convention for arrays?

Reply:
I'm actually not sure if it applies to this line, but there are several places where `tokenData` names an array of tokenDatas

Reply:
Renamed